### ISSUE TITLE
LocalizedClassList() broken on MAC client

### ISSUE THREAD

**Issue by zaphon** [Stanzilla/WoWUIBugs#521]
With patch 10.2.5 there was a change to the way the constant tables LOCALIZED_CLASS_NAMES_MALE and LOCALIZED_CLASS_NAMES_FEMALE get filled. The diff looks like this.

```
- LOCALIZED_CLASS_NAMES_MALE = {};
- LOCALIZED_CLASS_NAMES_FEMALE = {};
- FillLocalizedClassList(LOCALIZED_CLASS_NAMES_MALE, false);
- FillLocalizedClassList(LOCALIZED_CLASS_NAMES_FEMALE, true);
+ LOCALIZED_CLASS_NAMES_MALE = LocalizedClassList(false);
+ LOCALIZED_CLASS_NAMES_FEMALE = LocalizedClassList(true);

```

Now the issue is that on a PC, these two tables are filled in properly and contain all the data. But on a Mac, if you dump the tables or dump the actual call that creates the tables, it just returns the following.

```
Dump: value=LocalizedClassList(false);
[1]={
    [true]=“Warrior”
}

```

At least for me, quite a few addons rely on these tables (or the parent call to create their own table), and thus break on a Mac.

I have tested this with no addons loaded, as well as even tested it on the PTR.

**Comment by zaphon** [comment]
Oh and this is what the table should look like. This is from a Windows PC.

```
Dump: value=LocalizedClassList(false);
[1]={
  HUNTER="Hunter",
  WARRIOR="Warrior",
  PALADIN="Paladin",
  MAGE="Mage",
  DEATHKNIGHT="Death Knight",
  PRIEST="Priest",
  SHAMAN="Shaman",
  ROGUE="Rogue",
  WARLOCK="Warlock",
  DEMONHUNTER="Demon Hunter",
  MONK="Monk",
  DRUID="Druid",
  EVOKER="Evoker",
  Adventurer="Adventurer"
}
```

**Comment by Nevcairiel** [comment]
This is a known issue and Blizzard is actively working on fixing it soon.

**Comment by Meorawr** [comment]
Should be fixed as of either 10.2.5.52968 or 10.2.5.52983.
